kano_settings/system/advanced.py

```diff
@@ -368,5 +368,14 @@ def set_parental_level(level_setting):
         set_ultimate_parental(False)
 
     blacklist, whitelist = read_listed_sites()
+
+    # SafeSearch: disabling google.com, until we have higher Chromium version
+    # that implements policy files.
+    if 'chromium' in enabled:
+        if blacklist:
```

Comment:
I don't think we should do this... 

There's a function to write to the blacklist:
https://github.com/KanoComputing/kano-settings/blob/safe-search/kano_settings/system/advanced.py#L339

also blacklist should exist, if it doesn't then we have a problem we shouldn't just create the list with only 'google.com'

Also we need add more:
www.google.com
google.com
www.google.uk
google.uk
www.google.es
google.es
www.google.de
google.de

and so on...